apps/explorer/src/pages/AddressInfoPage/AddressTransactionRow.tsx

```diff
@@ -54,7 +54,7 @@ const AddressTransactionRow = ({ transaction: tx, addressHash, isInContract }: A
   const { assets, infoType } = useTransactionInfo(tx, addressHash)
 
   const isMoved = infoType === 'move'
-  const isPending = isMempoolTx(tx)
+  const isPending = !isConfirmedTx(tx)
```

Comment:
the following diffs you'll notice that I am using 
`Transaction | PendingTransaction | MempoolTransaction`, even though the last 2 are basically the same with the difference that `PendingTransaction` includes a `type: string` field. Another difference is that the `AcceptedTransaction | PendingTransaction` is a return value of the `GET /transactions/{transactionHash}` endpoint whereas `MempoolTransaction[]` is the return value of the mempool endpoint.

It'd be nice if we could clean these up, but until then this is my proposed workaround.

Reply:
yep. I agree it's still a bit confusing.

Reply:
Should we add a TODO somewhere?

Reply:
Yeah, probably in the shared package. Added here: bab402e50